BUILDING.md

```diff
@@ -8,6 +8,21 @@ You can skip to the [setup guide for Kubernetes](QUICKSTART-EKS.md) or the [setu
 
 ### Dependencies
 
+#### Adequate Disk Space
+
+The build process artifacts and resulting images can consume in excess of 4GB in the local directory.
+
+#### Operating System Packages
```

Comment:
Maybe call this "Linux" since it'll live at the same level as "Rust" and "Docker". Or we can fold it into the previous `System Requirements` section.